tests/connectors.yml

```diff
@@ -11,4 +11,4 @@ connector_id: "mongo"
 service_type: "mongo"
 log_level: info
 
-idle_timeout: 60
+poll_interval: 60
```

Comment:
I think we agreed on 5 seconds, right?

Reply:
I actually don't know what is under the dir `tests`, but I will update it.